processor/resourcedetectionprocessor/internal/system/metadata.yaml

```diff
@@ -19,6 +19,10 @@ resource_attributes:
     description: The os.type
     type: string
     enabled: true
+  os.version:
+    description: The os.version
+    type: string
+    enabled: false
```

Comment:
It is confusing for me as to why some of these attributes are enabled by default and why others are not.

I found https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21482 which adds some context for atleast `host.id` which is reasonable.

IMO though, the `os.version` though reasonably should be enabled here by default but I'm okay with the earlier suggestion of making this enabled by default as a separate change that we mark as breaking in the changelog.

Reply:
think of this also as a way to test things out for some users before we turn this on by default. This is all the more true for metrics, resource attributes can create time series recreations which can be painful if users are not aware of it.